src/fc/qemu/main.py

```diff
@@ -248,7 +248,13 @@ def main():
     except Exception:
         log.exception("unexpected-exception", exc_info=True)
         exitcode = 69  # EX_UNAVAILABLE
-    finally:
-        if exitcode != 0:
-            log.debug("exit", status=exitcode)
-        sys.exit(exitcode)
+
```

Comment:
you noticed this, too: let's catch systemexit here explicitly so we can log the correct exit code!